src/components/tables/RolesTable.vue

```diff
@@ -49,15 +55,16 @@ const columns = ref([{
   field: 'type',
   classes: 'role-type',
 }]);
-const roles = ref([]);
+const roleList = ref([]);
```

Comment:
useless you can have use props in template section.

Reply:
remove